com.valvesoftware.Steam.json

```diff
@@ -96,6 +96,9 @@
         {
             "name": "lspci",
             "no-autogen": true,
+	    "make-args": [
```

Comment:
Tab/spaces mix here.
```suggestion
            "make-args": [
```

Reply:
Sure, that's not the reason why this doesn't work though.

Reply:
Just noticed that and commented, sorry.

Reply:
Yeah, it's a good catch. I'm using Emacs and it wants to always put tabs into JSON. Anyhow, completely rewritten, I couldn't manage to get CFLAGS to go through without buildsystem simple (lspci isn't really an autotools project anyway, it just has a Makefile)